### DISCUSSION TITLE
Why not export `LatLng` in this package?

### DISCUSSION THREAD

**Discussion by buraktabn** [xclud/flutter_map#50] [Q&A]
It's a great package! I see that in order to set the center I need to use your `LatLng` package to set it. Which, I have to add a new package. It would be easy to just export `LatLng` in this package as well.

**Comment by xclud** [comment]
It's not only the LatLong class. The initial plan was to add even more utilities in that package. It's planned to be a full geodesic package but my time is limited now. Also, more packages may need to depend on LatLong but they do not need a map.
